src/AstroImages.jl

```diff
@@ -92,17 +106,19 @@ function AstroImage(file::String)
 end
 
 # Lazily render the image as a Matrix{Color}, upon request.
-function render(img::AstroImage{T,C}) where {T,C}
-    imgmin, imgmax = extrema(img.data)
+function render(img::AstroImage{T,C,N}, header_number = 1) where {T,C,N}
+    imgmin, imgmax = extrema(img.data[header_number])
     # Add one to maximum to work around this issue:
     # https://github.com/JuliaMath/FixedPointNumbers.jl/issues/102
     f = scaleminmax(_float(imgmin), _float(max(imgmax, imgmax + one(T))))
-    return colorview(C, f.(_float.(img.data)))
+    return colorview(C, f.(_float.(img.data[header_number])))
 end
 
 Images.colorview(img::AstroImage) = render(img)
 
-Base.size(img::AstroImage) = Base.size(img.data)
+Base.size(img::AstroImage) = Base.size.(img.data)
+
+Base.length(img::AstroImage) = Base.length(img.data)
```

Comment:
`size` is broadcasted, why `length` isn't?  Isn't this definition just equivalent to
```julia
Base.length(img::AstroImage{T,C,N}) where {T,C,N} = N
```
?

Also, same question as above: do you have a concrete case where this definition would be useful?

Reply:
> ```julia
> Base.length(img::AstroImage{T,C,N}) where {T,C,N} = N
> ```
Yeah, it can be written this way too.

I wanted to add something which tells users how many Images it holds(length) and what shape they each are (size). That's why I used broadcast to print out shape of each Image. Previously I found the same behaviour of checking shape of every other thing separately a bit inconvenient, that's why I added this way.
What are your suggestions?